source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -118,7 +118,12 @@ export abstract class CardElement {
         }
 
         if (this._separatorElement) {
-            this._separatorElement.style.display = this._isVisible ? this._defaultRenderedElementDisplayMode : "none";
+            if (this.parent && this.parent.isFirstElement(this)) {
+                this._separatorElement.style.display = "none";                
+            }
+            else {
+                this._separatorElement.style.display = this._isVisible ? this._defaultRenderedElementDisplayMode : "none";
+            }
```

Comment:
Could simplify into one conditional:

```
let shouldShowSeparator = this._isVisible && !(this.parent && this.parent.isFirstElement(this));
```
(or something similar).

Reply:
Actually I think that would make it harder to read and understand, which is why I did it in a more explicit way.